ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -133,20 +133,6 @@ public HelixClusterManager(ClusterMapConfig clusterMapConfig, String instanceNam
           initializationFailureMap.putIfAbsent(initializer.getDcName(), e);
         }
       }
-
-      for (DcInfo dcInfo : dcToDcInfo.values()) {
-        if (dcInfo.clusterChangeHandler instanceof ClusterMapChangeListener) {
```

Comment:
Why remove this code?  This code seems like generic clusterChangeListener that's not cloud DC specific.

Reply:
@panchow  Based on comments below looks this section is "tailored to the CloudServiceClusterChangeHandler, which only needs to be provided one replica per partition"

Reply:
Yeah, above code is sort of tailored for `CloudServiceClusterChangeHandler`.